deployment/CP03ISSP_Deploy.csv

```diff
@@ -1,37 +1,37 @@
+SI20150722,,AT31,,CP03ISSP-SP001-02-OPTAAJ000,3,1,2015-07-23T11:45:00,2015-10-15T20:12:00,CGMCP-03ISSP-00003,CGVEH-CSPPCP-00004,CGINS-OPTAAJ-00145,40.36437,-70.88813,,79,95,
+SI20150722,,AT31,,CP03ISSP-SP001-03-NUTNRJ000,3,1,2015-07-23T11:45:00,2015-10-15T20:12:00,CGMCP-03ISSP-00003,CGVEH-CSPPCP-00004,CGINS-NUTNRJ-00427,40.36437,-70.88813,,79,95,
+SI20150722,,AT31,,CP03ISSP-SP001-05-VELPTJ000,3,1,2015-07-23T11:45:00,2015-10-15T20:12:00,CGMCP-03ISSP-00003,CGVEH-CSPPCP-00004,CGINS-VELPTJ-11916,40.36437,-70.88813,,79,95,
+SI20150722,,AT31,,CP03ISSP-SP001-06-DOSTAJ000,3,1,2015-07-23T11:45:00,2015-10-15T20:12:00,CGMCP-03ISSP-00003,CGVEH-CSPPCP-00004,CGINS-DOSTAJ-00350,40.36437,-70.88813,,79,95,
+SI20150722,,AT31,,CP03ISSP-SP001-07-SPKIRJ000,3,1,2015-07-23T11:45:00,2015-10-15T20:12:00,CGMCP-03ISSP-00003,CGVEH-CSPPCP-00004,CGINS-SPKIRJ-00263,40.36437,-70.88813,,79,95,
+SI20150722,,AT31,,CP03ISSP-SP001-08-CTDPFJ000,3,1,2015-07-23T11:45:00,2015-10-15T20:12:00,CGMCP-03ISSP-00003,CGVEH-CSPPCP-00004,CGINS-CTDPFJ-00348,40.36437,-70.88813,,79,95,
+SI20150722,,AT31,,CP03ISSP-SP001-09-FLORTJ000,3,1,2015-07-23T11:45:00,2015-10-15T20:12:00,CGMCP-03ISSP-00003,CGVEH-CSPPCP-00004,CGINS-FLORTJ-01208,40.36437,-70.88813,,79,95,
+SI20150722,,AT31,,CP03ISSP-SP001-10-PARADJ000,3,1,2015-07-23T11:45:00,2015-10-15T20:12:00,CGMCP-03ISSP-00003,CGVEH-CSPPCP-00004,CGINS-PARADJ-00407,40.36437,-70.88813,,79,95,
+AR4,,AR4,,CP03ISSP-SP001-00-SPPENG000,4,1,2016-05-28T19:07:00,2016-05-30T16:44:00,CGMCP-03ISSP-00004,CGVEH-CSPPCP-00004,CGCON-CSPPCP-00004,40.36725,-70.88177,,79,94,
+AR4,,AR4,,CP03ISSP-SP001-02-OPTAAJ000,4,1,2016-05-28T19:07:00,2016-05-30T16:44:00,CGMCP-03ISSP-00004,CGVEH-CSPPCP-00004,CGINS-OPTAAJ-00145,40.36725,-70.88177,,79,94,
+AR4,,AR4,,CP03ISSP-SP001-03-NUTNRJ000,4,1,2016-05-28T19:07:00,2016-05-30T16:44:00,CGMCP-03ISSP-00004,CGVEH-CSPPCP-00004,CGINS-NUTNRJ-00427,40.36725,-70.88177,,79,94,
+AR4,,AR4,,CP03ISSP-SP001-05-VELPTJ000,4,1,2016-05-28T19:07:00,2016-05-30T16:44:00,CGMCP-03ISSP-00004,CGVEH-CSPPCP-00004,CGINS-VELPTJ-11916,40.36725,-70.88177,,79,94,
+AR4,,AR4,,CP03ISSP-SP001-06-DOSTAJ000,4,1,2016-05-28T19:07:00,2016-05-30T16:44:00,CGMCP-03ISSP-00004,CGVEH-CSPPCP-00004,CGINS-DOSTAJ-00350,40.36725,-70.88177,,79,94,
+AR4,,AR4,,CP03ISSP-SP001-07-SPKIRJ000,4,1,2016-05-28T19:07:00,2016-05-30T16:44:00,CGMCP-03ISSP-00004,CGVEH-CSPPCP-00004,CGINS-SPKIRJ-00263,40.36725,-70.88177,,79,94,
+AR4,,AR4,,CP03ISSP-SP001-08-CTDPFJ000,4,1,2016-05-28T19:07:00,2016-05-30T16:44:00,CGMCP-03ISSP-00004,CGVEH-CSPPCP-00004,CGINS-CTDPFJ-00348,40.36725,-70.88177,,79,94,
+AR4,,AR4,,CP03ISSP-SP001-09-FLORTJ000,4,1,2016-05-28T19:07:00,2016-05-30T16:44:00,CGMCP-03ISSP-00004,CGVEH-CSPPCP-00004,CGINS-FLORTJ-01208,40.36725,-70.88177,,79,94,
+AR4,,AR4,,CP03ISSP-SP001-10-PARADJ000,4,1,2016-05-28T19:07:00,2016-05-30T16:44:00,CGMCP-03ISSP-00004,CGVEH-CSPPCP-00004,CGINS-PARADJ-00407,40.36725,-70.88177,,79,94,
```

Comment:
looks good.  There are sensor cal csvs for sensors on WLP-004 for 2014 and 2016 in github.  We only have 2014 cals in Vault.  Need the 2016 cals since the sensors appear to have been sent back for refurb and recal prior to deployment 4.

Reply:
Looks good.